specs/latest/2.0/index.html

```diff
@@ -699,12 +700,18 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, ByteBuffer? data, GLenum usage,
+                  optional unsigned long byteOffset = 0);
+  void bufferSubData(GLenum target, GLintptr offset, ByteBuffer data,
+                     optional unsigned long byteOffset = 0);
+
   void copyBufferSubData(GLenum readTarget, GLenum writeTarget, GLintptr readOffset,
                          GLintptr writeOffset, GLsizeiptr size);
   // MapBufferRange, in particular its read-only and write-only modes,
   // can not be exposed safely to JavaScript. GetBufferSubData
   // replaces it for the purpose of fetching data back from the GPU.
-  void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData);
+  void getBufferSubData(GLenum target, GLintptr offset, ByteBuffer returnedData,
+                        optional unsigned long byteOffset = 0);
```

Comment:
It does, in order to still produce GL errors if the argument is missing (as opposed to IDL or DOM exceptions). Also, this should take a "size" argument as well.